website/public/js/controllers/notificationCtrl.js

```diff
@@ -70,23 +72,30 @@ habitrpg.controller('NotificationCtrl',
         User.user.items[type][after.key]++;
       }
 
-      if(after.type === 'HatchingPotion'){
-        var text = Content.hatchingPotions[after.key].text();
-        var notes = Content.hatchingPotions[after.key].notes();
-        Notification.drop(env.t('messageDropPotion', {dropText: text, dropNotes: notes}), after);
-      }else if(after.type === 'Egg'){
-        var text = Content.eggs[after.key].text();
-        var notes = Content.eggs[after.key].notes();
-        Notification.drop(env.t('messageDropEgg', {dropText: text, dropNotes: notes}), after);
-      }else if(after.type === 'Food'){
-        var text = Content.food[after.key].text();
-        var notes = Content.food[after.key].notes();
-        Notification.drop(env.t('messageDropFood', {dropArticle: after.article, dropText: text, dropNotes: notes}), after);
-      }else{
-        // Keep support for another type of drops that might be added
-        Notification.drop(User.user._tmp.drop.dialog);
+      switch(after.type) {
```

Comment:
Ah, if that's a bad pattern, I can revert it. I find it more readable than an `else if` chain, but I'm just a scrub, heh.